### ISSUE TITLE
[Accessibility- Script C 7.1] Lack of labels for some buttons

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3832]
### Problem Statement

Environment: 7.x

### Bug

**Step of reproduction:**
Add myApplications in a page

**Current bevavior**

The buttons:

- See all

- Edit Settings

don't have labels

**Expected behavior:**

All buttons listed above must have labels:

- Delete custom tooltip on the buttons

- Add the attribute title:

1. title="See All"

1. title="Edit Settings"
